agoradatatools/etl/transform.py

```diff
@@ -352,7 +365,8 @@ def apply_custom_transformations(datasets: dict, dataset_name: str, dataset_obj:
     elif dataset_name == "distribution_data":
         return transform_distribution_data(datasets=datasets,
                                            overall_max_score=dataset_obj['custom_transformations']['overall_max_score'],
-                                           genetics_max_score=dataset_obj['custom_transformations']['genetics_max_score'],
+                                           genetics_max_score=dataset_obj['custom_transformations'][
+                                               'genetics_max_score'],
```

Comment:
This is odd... can you put it all back on one line?

Reply:
done!